document-store/src/main/java/org/hypertrace/core/documentstore/mongo/query/parser/AliasParser.java

```diff
@@ -0,0 +1,37 @@
+package org.hypertrace.core.documentstore.mongo.query.parser;
+
+import org.hypertrace.core.documentstore.expression.impl.AggregateExpression;
+import org.hypertrace.core.documentstore.expression.impl.ConstantExpression;
+import org.hypertrace.core.documentstore.expression.impl.ConstantExpression.DocumentConstantExpression;
+import org.hypertrace.core.documentstore.expression.impl.FunctionExpression;
+import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
+import org.hypertrace.core.documentstore.parser.SelectTypeExpressionVisitor;
+import org.hypertrace.core.documentstore.parser.SortTypeExpressionVisitor;
+
+public class AliasParser implements SelectTypeExpressionVisitor, SortTypeExpressionVisitor {
+
+  @Override
+  public String visit(AggregateExpression expression) {
+    return null;
```

Comment:
Please consider returning an `Optional` instead of `null` and check in the clients.